rosidl_generator_py/resource/_msg_support.c.template

```diff
@@ -52,10 +48,22 @@ full_classname = "@(spec.base_type.pkg_name).@(subfolder)._@(module_name).@(spec
   PyObject * py@(field.name) = PyObject_GetAttrString(_pymsg, "@(field.name)");
   (void)py@(field.name);
 @[if not field.type.is_primitive_type()]@
-// TODO(esteve): add support for nested types
-@[end if]@
-
+@{
+nested_type = '%s__%s__%s' % (field.type.pkg_name,'msg',field.type.type)
+}@
 @[if field.type.is_array]@
+  // TODO implement nested array
```

Comment:
can you add TODO(<username>) here and elsewhere? It's a small style thing to make sure someone is associated with a hanging issue, rather than just letting issues hidden in the source code disappear. This normally is caught by cpplint, but the generated code doesn't get linted.